importlib_metadata/__init__.py

```diff
@@ -486,10 +495,35 @@ def _read_files_distinfo(self):
         text = self.read_text('RECORD')
         return text and text.splitlines()
 
-    def _read_files_egginfo(self):
+    def _read_files_egginfo_installed(self):
+        """
+        installed-files.txt might contain literal commas, so wrap
+        each line in quotes. Also, the entries in installed-files.txt
```

Comment:
This comment about 'literal commas' was okay when the implementation was small (barely different than _read_files_distinfo), but now that this interface is expanding to new use-cases, the description should probably give more precision around "returning CSV lines from installed-files.txt similar to lines from RECORD".

Reply:
Fixed in https://github.com/python/importlib_metadata/pull/437/commits/33eb7b4a0312017048d849cf1aef0321282d329d